janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLKeyColumnValueStore.java

```diff
@@ -426,8 +426,9 @@ public KeyIterator getKeys(final SliceQuery query, final StoreTransaction txh) t
         private int index;
         private int paginatedResultSize;
         private final Supplier<Statement> statementSupplier;
+        private AtomicLong recordCount = new AtomicLong();
```

Comment:
Why did you done these changes?

Reply:
I was trying to keep track of how many CQL records being read from DB and trying to keep a custom metric for the same. I  forgot removing counter after I removed that part the code.